recipes/proj-data/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.0" %}
+
+package:
+  name: proj-data
+  version: {{ version }}
+
+source:
+  url: http://download.osgeo.org/proj/proj-data-{{ version }}.tar.gz
+  sha256: 2637183bb2fa6cd421085734e5212b0c12550b8d5d3ecaadbfc1044648c7a906
+  folder: data
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - proj >=7.0.0
+
+test:
+  commands:
+    # TODO: we need real tests for each datumgrid.
+    - echo -105 40 | cs2cs +init=epsg:4326 +to +init=epsg:2975
+
+about:
+  home: https://github.com/OSGeo/proj-data
+  license: Public domain | X/MIT | BSD-2/3/4-Clause | CC0 | CC-BY (v3.0 or later) | CC-BY-SA (v3.0 or later)
+  # Each grid has a *README.txt file with the license info, that is copied with
+  # the datum files.
+  # license_file: *README.txt
+  summary: Data for Cartographic Projections and Coordinate Transformations Library
+
+extra:
+  recipe-maintainers:
+    - ocefpaf
+    - xylar
+    - snowman2
```

Comment:
@ocefpaf and @snowman2, are you willing to be maintainers? Are there others I should add?